Polyhedron/demo/Polyhedron/Scene_points_with_normal_item.cpp

```diff
@@ -711,17 +716,25 @@ QMenu* Scene_points_with_normal_item::contextMenu()
       {
         QMenu *container = new QMenu(tr("Normals Length"));
         QWidgetAction *sliderAction = new QWidgetAction(0);
-        connect(d->normal_Slider, &QSlider::valueChanged, this, &Scene_points_with_normal_item::invalidateOpenGLBuffers);
-        connect(d->normal_Slider, &QSlider::valueChanged, this, &Scene_points_with_normal_item::itemChanged);
-
+        if((d->nb_points + d->nb_selected_points)/3 <= 300000)
```

Comment:
I do not like magic constants! But know the constant is even hard-coded twice! Could it be a `const size_t` at the beginning of the file, so that it is at least easy to change?